### ISSUE TITLE
create new npm version containing the prisma.yml fix

### ISSUE THREAD

**Issue by danny-p93** [graphql-nexus/nexus-plugin-prisma#384]
As I started working with Nexus-Prisma-generate i was not able to configure the CLI to generate Nexus correctly with my Prisma.yml.
The Problem is already solved since May but is not published to npm, so npx and npm just found the old version that ignores the path to Prisma.yml

**Comment by jasonkuhrt** [comment]
Hi @danny-p93  what tool are you referring to exactly?

**Comment by danny-p93** [comment]
Nexus-Prisma-generate

when i run this command in my CLI i get the error Prisma.yml could not be found but of course the path is correct and the file exists:
`npx nexus-prisma-generate --project ./src/prisma/prisma.yml --client ./src/generated/prisma-client/  --output ./src/generated/nexus --js`

The Bug is already fixed with that issue/PR -> #272 but is not yet available in npm

**Comment by jasonkuhrt** [comment]
@danny-p93 thanks for linking to that issue. I understand now! :)

Done:

```
nexus-prisma@0.3.8 | MIT | deps: none | versions: 43
https://github.com/prisma/nexus-prisma-plugin#readme

dist
.tarball: https://registry.npmjs.org/nexus-prisma/-/nexus-prisma-0.3.8.tgz
.shasum: 19a94663903c7af17d63873476baeff845689d3c
.integrity: sha512-F+i6nKnT+a7RzvnYTwBTV5V6mW+euYm7nUbQUKw1DRJSWQfWbrHG/Jh3y0fx6iz6nIb8bDY9AozqILwNlhxFXA==
.unpackedSize: 91.6 kB

maintainers:
- schickling <schickling.j@gmail.com>

dist-tags:
beta: 0.0.1-beta.7  latest: 0.3.8

published 2 minutes ago by jasonkuhrt <jasonkuhrt@me.com>
```
